src/axom/inlet/Field.hpp

```diff
@@ -79,210 +79,39 @@ class Field : public VerifiableScalar
    */
   const axom::sidre::Group* sidreGroup() const { return m_sidreGroup; };
 
-  /*!
```

Comment:
Are you removing the doxygen comments b/c they are duplicates of what is already in `VerifiableScalar` ?

Reply:
Yes - it doesn't affect the generated documentation (probably would have been good to mention this in the PR description)